Add a blog post about Brontie (#56)
knpuniversity/blog: brontie-your-ai-companion.md

# Meet Brontie, your AI companion

We've been quietly working on something fun, and today's the day: meet **Brontie** —
your new SymfonyCasts companion. Part dinosaur, part AI, fully obsessed with Symfony.

You'll find Brontie hanging out in the comments of every tutorial — ready to help
when you get stuck on a tricky chapter, can't remember which
service to autowire, or just want a second opinion on your code.

> # How to summon Brontie

Reviewer: And let's do the same with sub-headers:
```suggestion
# How to Summon Brontie
```

Brontie lives in comments. To get her attention, mention `@Brontie` in any comment
on a chapter, course, or blog post — same as you'd tag a teammate.

That's it! Brontie will pop in with an answer shortly after — usually within

> about 15 seconds. Want to keep the conversation going? Just reply to her comment

Reviewer: hm, I'm thinking maybe we should not mention that time at all? Too specific, I'm thinking about `... shortly after. Want to keep the conversation...`

But up to you

Author: yea, I'll remove the specifics

like you would to anyone else — she remembers the thread.

> # Brontie has watched every tutorial on SymfonyCasts

Reviewer: ```suggestion
# Brontie has Watched Every Tutorial on SymfonyCasts
```

Here's the part we're really proud of: Brontie is trained on every SymfonyCasts

> tutorial. When you ask her a question, she pulls from the actual lessons we've written.

Reviewer: Probably best spot to mention that it also know Symfony docs/ libraries we're using, etc... what Kevin suggested here: https://github.com/knpuniversity/blog/pull/56#issuecomment-4480401002

Author: agree!

That means Brontie's answers stay grounded in the version of Symfony you're
learning — so she's far less likely to reach for a phantom bundle, a method that
was removed several versions ago, or an invented API. She's an AI, so she
won't be perfect, but the tutorials keep her honest.

> # Help Brontie get smarter

Reviewer: ```suggestion
# Help Brontie get Smarter
```

Every answer Brontie gives has a rating widget. If she nailed it, let her know.
If she missed the mark, tell us what went wrong — we read the feedback and use it
to sharpen Brontie up over time.

> # Joining the beta

Reviewer: ```suggestion
# Joining the Beta
```

remembers the thread.

# Brontie has Watched Every Tutorial on SymfonyCasts

Here's the part we're really proud of: Brontie is trained on every SymfonyCasts
tutorial. When you ask a question, Brontie pulls from the actual lessons we've
written.

Brontie has also been studying the official Symfony documentation and source
code, Doctrine, EasyAdmin, PHPUnit docs, and all the other libraries we cover
throughout our courses — so when Brontie points you to a method or feature,
it's grounded in the real ecosystem you're working in.

That means Brontie's answers stay closer to the version of Symfony you're
learning — and far less likely to reach for a method that was removed several versions ago,
or an invented API. Realstically, Brontie is an AI and it won't always be perfect, but the tutorials
and docs help keep Brontie as on track as a web developing dino can be.

# Help Brontie get Smarter

Every answer Brontie gives has a rating widget. If Brontie nailed it, say so.
If Brontie missed the mark, tell us what went wrong — we read the feedback and
use it to sharpen it up over time.

# Joining the Beta

Brontie is currently in closed beta for SymfonyCasts subscribers. We're letting

> people in gradually so we can keep an eye on quality and make sure it's is actually helping.

Author: ```suggestion
people in gradually, so we can keep an eye on quality and make sure it's actually helping.
```

# Meet Brontie, your AI companion

We've been quietly working on something fun, and today's the day: meet **Brontie** —
your new SymfonyCasts companion. Part dinosaur, part AI, fully obsessed with Symfony.

You'll find Brontie hanging out in the comments of every tutorial — ready to help
when you get stuck on a tricky chapter, can't remember which
service to autowire, or just want a second opinion on your code.

# How to summon Brontie

> Brontie lives in comments. To get her attention, mention @Brontie in any comment

Reviewer: Is it officially "she"? Should we use it/their instead? Or rewrite the sentence to avoid pronouns?

Also, let's tick the tag:
```suggestion
Brontie lives in comments. To get her attention, mention `@Brontie` in any comment
```

Author: thanks for bring it up. I'd like to hear @kbond and @Leannapelham oppinion about it too. For it is fine always mentioning the name `Brontie`

on a chapter, course, or blog post — same as you'd tag a
teammate.

> That's it! Brontie will pop in with an answer, usually within a few seconds.

Reviewer: I would use several here:

```suggestion
That's it! Brontie will pop in with an answer, usually within several seconds.
```

Author: it doesn't take that long, usually around 15 seconds

Want to keep the conversation going? Just reply to her comment
like you would to anyone else — she remembers the thread.

> # Brontie actually knows SymfonyCasts

Reviewer: Maybe:
```suggestion
# Brontie has watched every tutorial on SymfonyCasts
```

Here's the part we're really proud of: Brontie is trained on every SymfonyCasts
tutorial. When you ask her a question, she pulls from the
actual lessons we've written.

That means when Brontie shows you how to do something in Symfony, the code
matches the version of Symfony you're learning. No phantom bundles,
no "this method was removed in 5.4", no confidently inventing APIs that don't

> exist.

Reviewer: Hm, I didn't test it myself, but if this is really intended to work this way - it works to mention 👍🏼

Author: I can't guarantee it actually. Probably I should change this paragraph to avoid not hitting people's expectations

That means Brontie's answers stay closer to the version of Symfony you're
learning — and far less likely to reach for a method that was removed several versions ago,

> or an invented API. Realstically, Brontie is an AI and it won't always be perfect, but the tutorials

Reviewer: ?
```suggestion
or an invented API. Realistically, Brontie is an AI and it won't always be perfect, but the tutorials
```

# Help Brontie get smarter

Every answer Brontie gives has a rating widget. If she nailed it, let her know.
If she missed the mark, tell us what went wrong — we read the
feedback and use it to sharpen Brontie up over time.

# Joining the beta

Brontie is currently in closed beta for SymfonyCasts subscribers. We're letting
people in gradually so we can keep an eye on quality and make
sure Brontie is actually helping.

If you're a subscriber, head over to the [Brontie page](https://symfonycasts.com/brontie/beta) and request access.

> Happy chatting! 🦕

Reviewer: This blog post is missing metadata

# Meet Brontie

> We're excited to introduce you to our own Symfony trained AI helper: **Brontie**. 

Reviewer: IIRC we decided to call it assistant instead of helper?

Author: yep, AI coding assistant
